packages/calypso-e2e/src/lib/components/navbar-component.ts

```diff
@@ -6,6 +6,7 @@ const selectors = {
 	writeButton: '*css=a >> text=Write',
 	notificationsButton: 'a[href="/notifications"]',
 	meButton: 'a[data-tip-target="me"]',
+	closeCheckout: 'a[data-tip-target="close"]',
```

Comment:
This method should be implemented in https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-e2e/src/lib/pages/cart-checkout-page.ts instead, since the navbar is strictly only for items on the always-showing navbar.